### ISSUE TITLE
Lag for temporal reference pt and trend (number of data points)

### ISSUE THREAD

**Issue by jennifergriffiths** [OHI-Science/ohi-science.github.io#8]
This issue comes from working with the BHI project ECO calculation but I think it is generally applicable.

In general terms, our model is: Current value / Ref point.
Ref. point = value 5 years ago.

In our code we say the lag = 5.  Therefore if the current year = 2015.  The reference year = 2010.
For current year 2014, ref=2009  etc.

For the trend calculation, should there then be 6 data points? For the trend over the "previous 5 years," are we still saying current year(2015) minus 5?  Meaning that we would have six data points for the status from the year 2011 to 2015?

**Comment by Melsteroni** [comment]
Trend is typically calculated using 5 years of data points (e.g., 2010, 2011, 2012, 2013, 2014).  To establish trend years, we use the most recent year minus 4.

Thanks for pointing out the discrepancy in the manual, we will update this section!

It is good to be consistent with trend calculations (unless there is a good reason to deviate from the standard protocol), however because the trend estimates change in status per year the use of 5 vs. 6 years of data will not typically have a large effect (essentially it is similar to using a sample size of 5 vs. 6 to estimate an average).

**Comment by jennifergriffiths** [comment]
Thanks!  

_Quick follow-up_.  Okay. So we have 5 data points for the trend.  For a temporal reference point when we write "the reference point is the value five years ago"? would we still do: 
**ref_point_year = year - 5**

_Quick note_, the NP code in the BHI functions.r also is pretty explicit about having 6 data points (for 5 time intervals of a year) for trend. Not sure the origin of this code, but might be worth looking to see where else that occurs.  I'll look at the BHI code as I go.

**Comment by Melsteroni** [comment]
I agree:  If "the reference point is the value five years ago", then it
would be ref_point_year = year - 5
(the trend and reference point years aren't related so they do not need to
be the same)

Yes, the logic to use 5 time "intervals" (i.e., 6 data points) to calculate
trend crept in at one point, but it doesn't make sense because regression
models are calculated using data points and not intervals.  We have
eradicated this issue in the global analyses, but it sounds like there are
lingering cases.  Please let us know as you find them!

Thanks!

On Fri, Mar 25, 2016 at 7:18 AM, jennifergriffiths <notifications@github.com
